packages/truffle/test/sources/external_compile/truffle.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  compilers: {
+    external: {
+      command: "truffle compile " +
+        "--compiler=solc " +
+        "--contracts_build_directory=external",
+      targets: [{
+        path: "external/*.json",
+        command: "cat"
```

Comment:
What does `command` do? Is it a post-processing step?

Reply:
Yep - takes whatever files specified by `path` and pipes them to the `command`. The command is expected to output a JSON artifact on stdout.

Reply:
Sweet.